var/spack/repos/builtin/packages/py-bluepyemodel/package.py

```diff
@@ -9,11 +9,10 @@
 class PyBluepyemodel(PythonPackage):
     """Python library to optimize and evaluate electrical models."""
 
-    homepage = "https://bbpgitlab.epfl.ch/cells/bluepyemodel"
-    git = "ssh://git@bbpgitlab.epfl.ch/cells/bluepyemodel.git"
+    homepage = "https://github.com/BlueBrain/BluePyEModel"
+    pypi = "bluepyemodel/bluepyemodel-0.0.33.tar.gz"
```

Comment:
why not use latest version, 0.0.43?

Reply:
Does _not matter_ in the URI. This version string will get replaced by the stuff in `version(...)`